docker/nginx-proxy/conf/nginx.conf

```diff
@@ -0,0 +1,49 @@
+events {
+  worker_connections  4096;  ## Default: 1024
+}
+
+http {
+    server {
+    listen 80 default_server;
+    server_name localhost;
+
+        #
+        # Wide-open CORS config for nginx
+        #
+        location / {
+            if ($request_method = 'OPTIONS') {
+                add_header 'Access-Control-Allow-Origin' '*';
```

Comment:
If possible, pull this from an environment var or config? Or at least add a note that users may want to restrict this to the domain(s) that are hosting their client.